### PR TITLE
Demo environment update

### PR DESCRIPTION
**Pull request by lsenta**

### CODE REVIEW

**Comment by lsenta** [comment]
@Jackseed @GrandSchtroumpf can we merge this?

File: env/demo/env.demo1.ts [hunk 1]
```diff
@@ -1,6 +1,14 @@
 export const production = true;
 export const hmr = false;
 
+export const persistenceSettings = {
+  // TODO : change to synchronizeTabs when available (angularFire 5.2)
```
Comment (GrandSchtroumpf) [inline env/demo/env.demo1.ts]: Add an issue for this, and add the id of the issue in the comment.

File: env/demo/env.demo2.ts [hunk 2]
```diff
@@ -1,6 +1,14 @@
 export const production = true;
 export const hmr = false;
 
+export const persistenceSettings = {
+  // TODO : change to synchronizeTabs when available (angularFire 5.2)
```
Comment (GrandSchtroumpf) [inline env/demo/env.demo2.ts]: Same thing here.

File: env/demo/env.demo2.ts [hunk 3]
```diff
@@ -10,15 +18,31 @@ export const firebase = {
...
+// Ethereum
+// ========
 
-// DEMO VERSION: We use our staging setup to run the current production.
 export const network = 'goerli';
+export const mnemonic = ''; // defined in functions.config, see backend-functions/environments
 export const baseEnsDomain = 'blockframes.test';
+export const factoryContract = 'factory2.eth';
 
+// TODO : change the address
```
Comment (GrandSchtroumpf) [inline env/demo/env.demo2.ts]: Same here.

**Pull request merged**